coordinator/persistence/aggregation/src/integrationTest/kotlin/net/consensys/zkevm/persistence/dao/aggregation/AggregationsPostgresDaoTest.kt

```diff
@@ -469,6 +469,107 @@ class AggregationsPostgresDaoTest : CleanDbTestSuiteParallel() {
     }
   }
 
+  @Test
+  fun findHighestConsecutiveEndBlockNumber_returns_highest_consecutive_end_block_number_from_various_block_numbers() {
+    val aggregationProof1 = createProofToFinalize(
+      firstBlockNumber = 0,
+      finalBlockNumber = 10,
+      parentAggregationLastBlockTimestamp = Instant.fromEpochSeconds(0),
+      startBlockTime = Instant.fromEpochSeconds(0),
+      finalTimestamp = Instant.parse("2024-04-28T15:00:00Z")
+    )
+
+    val aggregation1 = Aggregation(
+      startBlockNumber = 1UL,
+      endBlockNumber = 10UL,
+      batchCount = 5UL,
+      aggregationProof = aggregationProof1
+    )
+    performInsertTest(aggregation1)
+
+    val aggregationProof2 = createProofToFinalize(
+      firstBlockNumber = 11,
+      finalBlockNumber = 20,
+      parentAggregationLastBlockTimestamp = Instant.parse("2024-04-28T15:00:00Z"),
+      startBlockTime = Instant.fromEpochSeconds(0),
+      finalTimestamp = Instant.parse("2024-04-28T15:01:00Z")
+    )
+
+    val aggregation2 = Aggregation(
+      startBlockNumber = 11UL,
+      endBlockNumber = 20UL,
+      batchCount = 5UL,
+      aggregationProof = aggregationProof2
+    )
+    performInsertTest(aggregation2)
+
+    val aggregationProof3 = createProofToFinalize(
+      firstBlockNumber = 21,
+      finalBlockNumber = 39,
+      parentAggregationLastBlockTimestamp = Instant.parse("2024-04-28T15:01:00Z"),
+      startBlockTime = Instant.fromEpochSeconds(0),
+      finalTimestamp = Instant.parse("2024-04-28T15:02:00Z")
+    )
+
+    val aggregation3 = Aggregation(
+      startBlockNumber = 31UL,
+      endBlockNumber = 39UL,
+      batchCount = 5UL,
+      aggregationProof = aggregationProof3
+    )
+    performInsertTest(aggregation3)
+
+    val aggregationProof4 = createProofToFinalize(
+      firstBlockNumber = 40,
+      finalBlockNumber = 50,
+      parentAggregationLastBlockTimestamp = Instant.parse("2024-04-28T15:01:00Z"),
+      startBlockTime = Instant.fromEpochSeconds(0),
+      finalTimestamp = Instant.parse("2024-04-28T15:02:00Z")
+    )
+
+    val aggregation4 = Aggregation(
```

Comment:
From what I noticed, start and end block numbers are the same between `aggregationProof` and `Aggregation`. It would be easier to read the test if we had a method that is generating an aggregation, inserting it into a db and returning an aggregation proof object